src/performance_counters/register_counters.cpp

```diff
@@ -46,6 +150,38 @@ namespace phylanx { namespace performance_counters
             &ir::node_data<double>::move_assignment_count,
             "returns the current value of the move-assignment count of "
                 "any node_data<double>");
+
+
+        namespace et = phylanx::execution_tree;
+
+        // Query the list of existing primitive types
+        et::pattern_list const& pattern_list = et::get_all_known_patterns();
+
+        // Iterate and register a time and count performance counter per each primitive
+        for (auto const& patterns : pattern_list)
+        {
+            // No need to process all entries in the list
+            auto const& pattern = *(patterns.begin());
+            // The name of the primitive
+            std::string const& name = hpx::util::get<0>(pattern);
+            // Register a primitive time performance counter
+            hpx::performance_counters::install_counter_type(
+                "/phylanx/time/primitives/" + name + "/eval",
+                hpx::performance_counters::counter_raw_values,
+                "returns the total execution time of eval() function of the " +
+                    name + " primitive",
+                &primitive_counter_creator,
+                &hpx::performance_counters::locality_counter_discoverer,
+                HPX_PERFORMANCE_COUNTER_V1, "ns");
+            // Register a primitive count performance counter
+            hpx::performance_counters::install_counter_type(
+                "/phylanx/count/primitives/" + name + "/eval",
+                hpx::performance_counters::counter_raw_values,
+                "returns the number of times of eval() function of each " +
```

Comment:
Could you fix the grammar of the help text, please?

Reply:
Updated. Check again, please.